src/helperFunctions/getCloudinaryImageUrl.ts

```diff
@@ -0,0 +1,18 @@
+const cloudinaryImageUrl = `https://res.cloudinary.com/realdevsquad/image/upload`;
+const userBaseUrl = `https://api.realdevsquad.com`;
```

Comment:
Can we follow the convention of naming variables with uppercase ? For example -DUMMY_IMAGE_URL which you have earlier used.

Reply:
Yeah, that would be better. Done that. Thanks